modules/core/include/opencv2/core/mat.hpp

```diff
@@ -371,6 +371,7 @@ class CV_EXPORTS _OutputArray : public _InputArray
     void release() const;
     void clear() const;
     void setTo(const _InputArray& value, const _InputArray & mask = _InputArray()) const;
+    Mat reinterpretType( int type ) const;
```

Comment:
can we skip `Type` suffix in the name? (`Mat reinterpret(int type) const;`?)

Reply:
@vpisarev I had done